tests/common.go

```diff
@@ -9369,3 +9425,34 @@ func AddCloudCredentialOwnership(cloudCredentialName string, cloudCredentialUid
 	}
 	return nil
 }
+
+// GenerateS3BucketPolicy Generates an S3 bucket policy based on encryption policy provided
+func GenerateS3BucketPolicy(sid string, encryptionPolicy string, bucketName string) (string, error) {
+
+	encryptionPolicyValues := strings.Split(encryptionPolicy, "=")
+	if len(encryptionPolicyValues) < 2 {
+		return "", fmt.Errorf("Failed to generate policy for s3 check for proper length of encryptionPolicy")
```

Comment:
mention the `encryptionPolicy` that the caller has passed

Reply:
done

Reply:
done